MovieViewer/MoviesViewController.swift

```diff
@@ -61,11 +60,14 @@ class MoviesViewController: UIViewController {
         }
     }
     var displayMode = DisplayMode.Grid
+    var ref: FIRDatabaseReference?
```

Comment:
I think you can set this to `!`. You'll always be initializing it in viewDidLoad.